hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/shortcircuit/ShortCircuitCache.java

```diff
@@ -379,13 +380,18 @@ public static ShortCircuitCache fromConf(ShortCircuitConf conf) {
   public ShortCircuitCache(int maxTotalSize, long maxNonMmappedEvictableLifespanMs,
       int maxEvictableMmapedSize, long maxEvictableMmapedLifespanMs,
       long mmapRetryTimeoutMs, long staleThresholdMs, int shmInterruptCheckMs) {
-    Preconditions.checkArgument(maxTotalSize >= 0);
+    Preconditions.checkArgument(maxTotalSize >= 0,
+      "Invalid argument: maxTotalSize must be greater than zero.");
```

Comment:
No need to put Invalid Argument:, It will throw IllegalArgumentException, that is indicative. just keep the message post Invalid argument: in all the Preconditions.